docs/topics/resources/spring-boot/spring-boot-openshift-pom-config-example.xml

```diff
@@ -0,0 +1,38 @@
+<profiles>
+    <profile>
+      <id>licenses</id>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.codehaus.mojo</groupId>
+            <artifactId>license-maven-plugin</artifactId>
+          </plugin>
+          <plugin>
+            <groupId>org.codehaus.mojo</groupId>
+            <artifactId>xml-maven-plugin</artifactId>
+          </plugin>
+        </plugins>
+      </build>
+    </profile>
+    <profile>
+      <id>openshift</id>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>io.fabric8</groupId>
+            <artifactId>fabric8-maven-plugin</artifactId>
+            <version>{Fabric8MavenPluginVersion}</version>
+            <executions>
+              <execution>
+                <id>fmp</id>
```

Comment:
This line should be removed. There's a `default` execution to which the `resource` and `build` goals are to be added.